### ISSUE TITLE
NOT AN ISSUE PERSE

### ISSUE THREAD

**Issue by finey-dev** [1RandomDev/mailspring-api#1]
I was wondering if your API implementation fetches emails and other details related to the email account, basically the whole authentication flow upto when the user is able to retrieve and send emails.

**Comment by 1RandomDev** [comment]
No, emails and email credentials are never sent to the server (they did in the past with Mailspring, but they changed it due to user concerns way before I created this project). All features that require sending emails like send later rely on a background service on your computer that Mailspring keeps running even after you close the programm. The API is mainly used for synchronizing that information between different devices and for features like link/open tracking.

**Comment by nouritsu** [comment]
@1RandomDev considering your fork does not have a section for issues (and this issue is basically a non issue) I had a question -

Does your fork of the repository just change the share URL OR does it add functionality which allows for changing the URL in the first place. Based on the last commit it does the former, but I may be mistaken.
